ethers-solc/src/config.rs

```diff
@@ -420,7 +437,8 @@ impl Default for ProjectPaths {
         Self {
             artifacts: "out".into(),
             sources: "src".into(),
-            tests: "tests".into(),
+            tests: "test".into(),
```

Comment:
I changed here, because it seems like `tests` was a mistake here. Should I revert it?

Reply:
that makes sense, is also the default of `foundry::Config`